src/main/java/com/cloudbees/jenkins/plugins/advisor/AdvisorGlobalConfiguration.java

```diff
@@ -211,8 +211,8 @@ public void setEmail(@CheckForNull String email) {
     return email;
   }
 
-  public @Nonnull String getPassword() {
-    return Secret.toString(password);
+  public @Nonnull Secret getPassword() {
```

Comment:
I think this change ends up removing the password from the password field after the configuration is saved.  We moved to the String in order to have that password appear.